CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 .. _changelog:
 
+0.12.2
+------
+
+Minor release to fix various bugs across the components.
```

Comment:
Please add one more sentence about the bugs which were fixed and/or the affected components.

Reply:
Done. Please take a look @ableuler.

Reply:
normally we'd link to a specific release or changelog from the individual components. I realize now this wasn't done for 0.12.1 either, so "minor bug fixes" is pretty ambiguous.

Reply:
Added links to releases.

Reply:
great, thanks!

Reply:
@olevski do we need to add a blurb about the problems with renku-core metrics?

Reply:
@rokroskar sorry, I've closed the PR already. I hope no more changes are needed...